src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--1 calendar__day--mon"></div>
```

Comment:
`--mon` modifier should be on `calendar`. also it should be possible to change quantity of days by modifier `calendar--28`. modifiers `calendar__day--1..31` are redundant